openmc/arithmetic.py

```diff
@@ -604,6 +604,10 @@ def bins(self):
     def num_bins(self):
         return len(self.bins) if self.aggregate_filter else 0
 
+    @property
+    def shape(self):
+        return (self.num_bins,)
```

Comment:
```suggestion
        return (self.num_bins)
```

Reply:
yes it is because we want `shape` to be a tuple. I think what you have here will just return the integer number of bins